packages/react-components/react-combobox/src/utils/useComboboxPopup.ts

```diff
@@ -0,0 +1,38 @@
+import { resolvePositioningShorthand, usePositioning } from '@fluentui/react-positioning';
+import { ExtractSlotProps, Slot, useMergedRefs } from '@fluentui/react-utilities';
+import type { ComboboxBaseProps } from './ComboboxBase.types';
+import { Listbox } from '../components/Listbox/Listbox';
+
+export function useComboboxPopup(
```

Comment:
nit: Naming is hard, but should not it be a popover to be consistent with `Popover` component? Or is it intentional?